retro/tables/pexp_5d.py

```diff
@@ -558,10 +598,12 @@ def grad(scalefactor):
 
         if scalefactor < 0:
             scalefactor = 0
+        if scalefactor > 1000:
+            scalefactor = 1000
 
         # -- Calculate the scale factor -- #
-
-        llh = -sum_nonscaling_exp_charge - scalefactor * sum_scaling_exp_charge
+        llh = np.float64(0)
+        llh += -sum_nonscaling_exp_charge - scalefactor * sum_scaling_exp_charge
```

Comment:
I don't understand why you would define as zero then add to it; if you want to make sure it's a float64, you can wrap this second line in `float64(...)`

Reply:
it was just testing...removed